apps/site/lib/site/application.ex

```diff
@@ -19,7 +21,8 @@ defmodule Site.Application do
       supervisor(Site.GreenLine.Supervisor, []),
       supervisor(Site.Stream.Vehicles, []),
       supervisor(Site.React, []),
-      supervisor(Site.RealtimeSchedule, [])
+      supervisor(Site.RealtimeSchedule, []),
+      supervisor(ConCache, [[], [name: :line_diagram_realtime_cache, ttl: :timer.seconds(60)]])
```

Comment:
Since your controller depends on this cache being available, the cache should be started before `SiteWeb.Endpoint`.